apps/frontend/src/components/RequestModal/RequestModal.tsx

```diff
@@ -66,10 +66,14 @@ export default function RequestModal({ show, handleCloseModal, request }: Props)
           as="textarea"
           rows={4}
           name="content"
+          className="mb-3"
           minLength={4}
           required
-          isInvalid={responseInput.trim().length < 4 && formSubmitted}
-          isValid={responseInput.trim().length >= 4}
+          isInvalid={
+            (responseInput.trim().length < 4 || !validTextAreaPattern.test(responseInput)) &&
```

Comment:
Why are we checking validTextAreaPattern. It's the same thing as responseInput.trim().length < 4

Reply:
It’s not exactly the same. The regex ensures that at least 4 non white space characters have been inputted. Eg `f   O` is invalid but `f   Ooo` is not.

Reply:
Understood! Then, I think we can remove `responseInput.trim().length < 4` since it's already checked by the Regex, right?